pyroll/core/roll/roll.py

```diff
@@ -50,6 +50,9 @@ class Roll(HookHost):
     temperature = Hook[float]()
     """Mean temperature."""
 
+    surface_temperature = Hook[float]()
+    """Effective surface temperature."""
```

Comment:
Effective?

Reply:
Means not a local value on a certain surface point.